src/Pate/Verification/StrongestPosts.hs

```diff
@@ -468,26 +470,68 @@ eqMemOp sym (MT.MemOp xptr xdir xcond xw xval xend) (MT.MemOp yptr ydir ycond yw
 
   | otherwise = return (W4.falsePred sym)
 
+
+data SeqTwoCache a b = SeqTwoCache (IORef (Map.Map (Maybe (Nonce GlobalNonceGenerator a), Maybe (Nonce GlobalNonceGenerator a)) b))
```

Comment:
I'm pretty sure I understand what this is doing, but the name of this type bothered me for reasons I can't articulate.

Can we call it something like a `SeqPairCache`?